packages/admin/src/targetGroups/TargetGroupForm.tsx

```diff
@@ -175,19 +176,36 @@ export function TargetGroupForm({ id, scope, additionalFormFields, input2State,
                             </FieldSet>
                         )}
                         {id && (
-                            <FieldSet
-                                title={
-                                    <FormattedMessage id="cometBrevoModule.targetGroup.manuallyAddContacts" defaultMessage="Manually add contacts" />
-                                }
-                                initiallyExpanded
-                                disablePadding
-                            >
-                                <AddContactsGridSelect
-                                    assignedContactsTargetGroupBrevoId={data?.targetGroup.assignedContactsTargetGroupBrevoId ?? undefined}
-                                    id={id}
-                                    scope={scope}
-                                />
-                            </FieldSet>
+                            <>
+                                <FieldSet
+                                    title={
+                                        <FormattedMessage
+                                            id="cometBrevoModule.targetGroup.manuallyAddContacts"
+                                            defaultMessage="Manually add contacts"
+                                        />
+                                    }
+                                    initiallyExpanded
+                                    disablePadding
+                                >
+                                    <AddContactsGridSelect
+                                        assignedContactsTargetGroupBrevoId={data?.targetGroup.assignedContactsTargetGroupBrevoId ?? undefined}
+                                        id={id}
+                                        scope={scope}
+                                    />
+                                </FieldSet>
+                                <FieldSet
+                                    title={
+                                        <FormattedMessage
+                                            id="cometBrevoModule.targetGroup.allAssignedContacts"
+                                            defaultMessage="All assigned contacts"
+                                        />
+                                    }
+                                    disablePadding
+                                    initiallyExpanded={false}
```

Comment:
not important but this is probably unnecessary, i guess initiallyExpanded={false} is the default, and you just set initiallyExpanded when you need it expanded